installer_scripts/install-voyager-airgapped.sh

```diff
@@ -183,35 +187,56 @@ install_perl_module() {
     # Return to the original directory
     cd ..
     
-    # Verification and version check
+    # Verification of the installed module
+    check_perl_module_version "$module_name" "$requirement_type" "$required_version" "false" 
```

Comment:
do we have such verification of other packages we install such as voyager, ora2pg etc..?